src/consensus/params.h

```diff
@@ -7,8 +7,29 @@
 #define BITCOIN_CONSENSUS_PARAMS_H
 
 #include "uint256.h"
+#include <map>
+#include <string>
 
 namespace Consensus {
+
+enum DeploymentPos
```

Comment:
I had the same confusion as @petertodd when initially reading the code, so for new developers it must be worse. I do think at least a comment should be added.